families/track_and_trade/client/src/views/fish_detail.js

```diff
@@ -38,6 +38,12 @@ const _labelProperty = (label, value) => [
     m('dd', value))
 ]
 
+const _row = (...cols) =>
+  m('.row',
+    cols
+    .filter((col) => col !== null)
```

Comment:
FYI, This essentially rewrites an existing function from `components/layout`

Reply:
Also, you don't have to filter out `null` values. Mithril already ignores those.

Reply:
It's a little different - different column type.